src/ImageRunModal.jsx

```diff
@@ -789,7 +789,7 @@
                                 onClear={this.clearImageSelection}
                                 // onFilter must be set or the spinner crashes https://github.com/patternfly/patternfly-react/issues/6384
                                 onFilter={() => {}}
-                                onTypeaheadInputChanged={value => this.debouncedInputChanged(value)}
+                                onTypeaheadInputChanged={(_, value) => this.debouncedInputChanged(value)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230606-080503-570e483c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)